client/dashboard/agency-overview/index.tsx

```diff
@@ -1,22 +1,23 @@
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { PageHeader } from '../components/page-header';
 import PageLayout from '../components/page-layout';
 
 export default function AgencyOverview() {
 	return (
-		<PageLayout
-			title={ __( 'Agency Overview' ) }
-			actions={
-				<>
-					<Button variant="primary" __next40pxDefaultSize>
+		<PageLayout>
+			<PageHeader
+				title={ __( 'Agency Overview' ) }
+				actions={ [
```

Comment:
Actions being an array forces you to have "keys", why don't we just make it a React element.

Reply:
Sounds good. I had it that way before that I had the `filter(Boolean)` baked in.